integration_tests/samples/socket_mode/websocket_client_example.py

```diff
@@ -0,0 +1,39 @@
+# ------------------
+# Only for running this script here
+import sys
+from os.path import dirname
+
+sys.path.insert(1, f"{dirname(__file__)}/../../..")
+# ------------------
+
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+import os
+from threading import Event
+from slack_sdk.web import WebClient
+from slack_sdk.socket_mode.response import SocketModeResponse
+from slack_sdk.socket_mode.request import SocketModeRequest
+from slack_sdk.socket_mode.websocket_client import SocketModeClient
+
+client = SocketModeClient(
+    app_token=os.environ.get("SLACK_SDK_TEST_SOCKET_MODE_APP_TOKEN"),
+    web_client=WebClient(token=os.environ.get("SLACK_SDK_TEST_SOCKET_MODE_BOT_TOKEN")),
```

Comment:
You can skip this if you pass token as parameter in API calls. This is just to give the default token to use for API calls.